legal-api/src/legal_api/services/queue.py

```diff
@@ -92,8 +92,11 @@ def init_app(self, app, loop=None,
 
     def teardown(self, exception):  # pylint: disable=unused-argument; flask method signature
         """Destroy all objects created by this extension."""
-        this_loop = asyncio.get_event_loop()
-        this_loop.run_until_complete(self.close())
+        try:
```

Comment:
Mildly concerned about this. I've done the same to get past this code during local dev, but what's the impact in production? @thorwolpert concerns?

Reply:
as discussed in person - approved, no issue with this change